src/plugins/data/server/query/route_handler_context.ts

```diff
@@ -6,6 +6,15 @@
  * Side Public License, v 1.
  */
 
+/*
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
```

Comment:
seems this is double licensed now
```suggestion
```

Reply:
there's a German saying "Doppelt genäht hält besser" but I think it doesn't apply here